lib/internal/errors.js

```diff
@@ -241,6 +243,7 @@ E('ERR_OUTOFMEMORY', 'Out of memory');
 E('ERR_STDERR_CLOSE', 'process.stderr cannot be closed');
 E('ERR_STDOUT_CLOSE', 'process.stdout cannot be closed');
 E('ERR_STREAM_WRAP', 'Stream has StringDecoder set or is in objectMode');
+E('ERR_STRING_WITH_LENGTH_0', '%s must be a string with length > 0');
```

Comment:
I think this would be better as `ERR_INVALID_STRING_LENGTH` and if it emulated `ERR_INVALID_ARRAY_LENGTH`.